test/integration_tests/abao/abao_test_hooks.js

```diff
@@ -48,6 +48,7 @@ hooks.skip("POST /download -> 200");
 hooks.skip("GET /download -> 200");
 hooks.skip("POST /upload/label -> 200");
 hooks.skip("POST /upload/uid -> 200");
+hooks.skip("POST /upload/uid -> 404");
```

Comment:
Can you add a comment on why this was skipped?  The others apply to the comment above about testing file fields, which now should probably say postman something